x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -30,9 +31,14 @@ export const ComplianceDashboard = () => {
     <CloudPosturePage query={getDashboardData}>
       <EuiPageHeader
         bottomBorder
-        pageTitle={i18n.translate('xpack.csp.dashboard.cspPageTemplate.pageTitle', {
-          defaultMessage: 'Cloud Posture',
-        })}
+        pageTitle={
```

Comment:
sidenote: looks like we're appending the icon to the right, which puts it in a different position depending on the page title width. we _could_ append to the left, keeping its position fixed AND using only `iconType` prop from `EuiPageHeader`

Reply:
good suggestion but some Kibana pages are already appending right and we better align to that